test/Makefile

```diff
@@ -25,6 +25,9 @@ ifneq ($(RECENT_GCC),"yes")
   FILES 	:= $(filter-out Debug.test,$(FILES))
 endif
 
+# See F* bug 2641 and associated PR
+FILES := $(filter-out Layered.test,$(FILES))
```

Comment:
Why not simply add this test to the list of broken tests 12 lines above? I will be fixing this in #271 since I noticed this while merging master in.